channels/sa.m3u

```diff
@@ -35,16 +31,10 @@ https://cdnamd-hls-globecast.akamaized.net/live/ramdisk/saudi_sunnah/hls1/saudi_
 https://iptv--iptv.repl.co/Arabic/Sunnah_TV/
 #EXTINF:-1 tvg-id="AlWesalTV.sa" tvg-name="Al Wesal TV" tvg-country="SA" tvg-language="Arabic" tvg-logo="" group-title="Religious",Al Wesal TV (720p)
 http://live.noorlive.com:1935/wesal/wesal1/playlist.m3u8
-#EXTINF:-1 tvg-id="AlResalah.sa" tvg-name="Al-Resalah" tvg-country="SA" tvg-language="Arabic" tvg-logo="" group-title="Religious",Al-Resalah (1080p)
-https://rotananet.hibridcdn.net/rotana/risala_abr/rotana/risala_1080p/chunks.m3u8
-#EXTINF:-1 tvg-id="AlResalah.sa" tvg-name="Al-Resalah" tvg-country="SA" tvg-language="Arabic" tvg-logo="" group-title="Religious",Al-Resalah (240p)
-https://rotananet.hibridcdn.net/rotana/risala_abr/rotana/risala_240p/chunks.m3u8
-#EXTINF:-1 tvg-id="AlResalah.sa" tvg-name="Al-Resalah" tvg-country="SA" tvg-language="Arabic" tvg-logo="" group-title="Religious",Al-Resalah (360p)
-https://rotananet.hibridcdn.net/rotana/risala_abr/rotana/risala_360p/chunks.m3u8
-#EXTINF:-1 tvg-id="AlResalah.sa" tvg-name="Al-Resalah" tvg-country="SA" tvg-language="Arabic" tvg-logo="" group-title="Religious",Al-Resalah (480p)
-https://rotananet.hibridcdn.net/rotana/risala_abr/rotana/risala_480p/chunks.m3u8
-#EXTINF:-1 tvg-id="AlResalah.sa" tvg-name="Al-Resalah" tvg-country="SA" tvg-language="Arabic" tvg-logo="" group-title="Religious",Al-Resalah (720p)
-https://rotananet.hibridcdn.net/rotana/risala_abr/rotana/risala_720p/chunks.m3u8
```

Comment:
Working links. Not duplicates.

Reply:
They don't work for me but I added them back